tutorials/cp-sdk-android-flows-passcode/cp-sdk-android-flows-passcode.md

```diff
@@ -0,0 +1,185 @@
+---
+author_name: Haoyue Feng
+author_profile: https://github.com/Fenghaoyue
+title: Handle Passcode with the Flows Component
+description: Learn how to use the Flows component of SAP BTP SDK for Android to handle passcode change, forgetting passcode and passcode verification scenarios.
+auto_validation: true
+time: 60
+tags: [ tutorial>beginner, operating-system>android, topic>mobile, products>sap-business-technology-platform]
+primary_tag: products>sap-btp-sdk-for-android
+---
+
+## Prerequisites
+- You completed [Try Out the SAP BTP SDK Wizard for Android](cp-sdk-android-wizard-app).
```

Comment:
@FengHaoyue Can we please add the minimum SDK version the student needs to have? If the version doesn't matter then we don't need to include anything.

Reply:
All the tutorials are always based on the latest version of the SDK. Once a new version is released, we update the tutorial.

Reply:
Thank you for the confirmation Haoyue.